base/loggers/Loggers.cpp

```diff
@@ -141,9 +168,17 @@ void Loggers::buildLoggers(Poco::Util::AbstractConfiguration & config, Poco::Log
     {
         bool color_enabled = config.getBool("logger.color_terminal", color_logs_by_default);
 
+        auto console_loglevel_string = config.getString("logger.console_loglevel", log_level_string);
```

Comment:
better ideas how to name the param? Or ok?

Reply:
I would prefer `console_log_level`, but not sure that it's better